dashboard/views.py

```diff
@@ -23,3 +24,13 @@ def get_context_data(self, **kwargs):
         context['domains'] = domains
         context['push_apps'] = push_apps
         return context
+
+
+class VapidValidation(TemplateView):
```

Comment:
It will probably be cleaner to re-name this 'PushApplication' and move it to the push app `push/views.py` ...

So It would essentially become the "Push Application Details" page that shows everything about the push app: Name, VAPID key, status, and validated.

If the Push Application is not validated, the details page would include a form for pasting & submitting the signed vapid_key_token to verify the push app.